composer.json

```diff
@@ -21,15 +21,21 @@
         }
     ],
     "require": {
-        "php": "^8.0",
-        "illuminate/contracts": "9.0 - 9.34 || ^9.36 || ^10.0",
-        "lorisleiva/lody": "^0.4"
+        "php": "^8.1",
+        "illuminate/contracts": "^10.0 || ^11.0",
+        "lorisleiva/lody": "dev-l11-compatibility"
```

Comment:
Lody needs to be upgraded first.

Reply:
Hi @lorisleiva 

I noted that in the commit message

```
Use Compatible Lody version

This commit can be removed and the correct version of lody use after https://github.com/lorisleiva/lody/pull/6 has been merged
```

I'm happy for you to use this branch or close it. Whatever is easier.

Thanks for your work.